src/status_im/contexts/centralized_metrics/tracking.cljs

```diff
@@ -3,37 +3,61 @@
     [legacy.status-im.utils.build :as build]
     [react-native.platform :as platform]))
 
-(defn user-journey-event
-  [action]
+(defn key-value-event
+  [event-name val-key value]
   {:metric
-   {:eventName  "user-journey"
+   {:eventName  event-name
     :platform   platform/os
     :appVersion build/app-short-version
-    :eventValue {:action action}}})
+    :eventValue {val-key value}}})
+
+(defn user-journey-event
+  [action]
+  (key-value-event "user-journey" :action action))
+
+(defn navigation-event
+  [view-id]
+  (key-value-event "navigation" :viewId view-id))
```

Comment:
Little code comment 🐭 
Could we pass the `val-key` and `value` as a map?
For example, we could call `key-value-event` like:
```clojure
(key-value-event "event-name" {:event-key event-value})
```
Perhaps if we do that we would rename `key-value-event` into `named-event` or maybe `metric-event`. Thoughts?

Reply:
It's better for sure. But I built it in a rush yesterday evening. 

We wanted to postpone it to 2.31, but Icaro suggested that getting onboarding in is not hard. 

This is only the first version. 2.31 will be way more elaborate.